src/main/scala/mesosphere/marathon/api/v2/InfoResource.scala

```diff
@@ -17,7 +21,7 @@ class InfoResource @Inject() (
     schedulerService: MarathonSchedulerService,
     mesosLeaderInfo: MesosLeaderInfo,
     leaderInfo: LeaderInfo,
-    conf: MarathonConf with HttpConf with EventConfiguration with HttpEventConfiguration with LeaderProxyConf) {
+    conf: MarathonConf with HttpConf with EventConfiguration with HttpEventConfiguration with LeaderProxyConf with OfferMatcherManagerConfig with OfferProcessorConfig with DebugConf with LaunchQueueConfig with LaunchTokenConfig) { // scalastyle:ignore
```

Comment:
How to configure autoformater to split this into multiple lines instead of joining into long one?